build.sh

```diff
@@ -90,7 +90,10 @@ install_rpms() {
 
     # Commented out for now, see above
     #dnf remove -y $builddeps}
-    rpm -q grubby && yum remove -y grubby
+    # can't remove grubby on el7 because libguestfs-tools depends on it
```

Comment:
Dang, I was going to suggest we should at least remove `kernel` on RHEL, but `libguestfs-tools-c` requires it.  :roll_eyes: